src/Vezeeta.Domain/Entities/Times.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Vezeeta.Domain.Entities
+{ 
+    public class Times
```

Comment:
For all entities, you should be inheriting from `BaseEntity<T>` where T is the datatype of the ID for your entity.